lib/datastore/query.js

```diff
@@ -75,7 +75,16 @@ function Query(namespace, kinds) {
 }
 
 /**
+ * @param {boolean=} autoPaginateVal - Have pagination handled automatically.
+ *     Default: true.
  * @return {module:datastore/query}
+ *
+ * @example
+ * // Have pagination handled automatically
+ * var autoPaginated = query.autoPaginate();
```

Comment:
Want to take a look at the updated example? The query is a little more complex, but I think it needs to be in order to properly display `autoPaginate()` in a non-foobar fashion.